src/app/helptext/system/advanced.ts

```diff
@@ -45,6 +46,14 @@ export const helptext_system_advanced = {
  DISCOURAGED</b>'),
   swapondrive_validation: [ Validators.required, Validators.min(0), Validators.max(99) ],
 
+  overprovision: {
+    placeholder: T('Overprovision Size in GiB - (Examples: 50 GiB, 50g, 50GB)'),
```

Comment:
Is this too long for a field title? The examples could be added to the help text instead.
```suggestion
    placeholder: T('Overprovision Size in GiB'),
```